torch/csrc/jit/tensorexpr/types.cpp

```diff
@@ -74,8 +74,15 @@ int Dtype::byte_size() const {
     scalar_size = sizeof(Type); \
     break;
 
-    AT_FORALL_SCALAR_TYPES_AND5(
-        Bool, Half, BFloat16, Float8_e5m2, Float8_e4m3fn, TYPE_CASE);
+    AT_FORALL_SCALAR_TYPES_AND7(
```

Comment:
Why is this needed here? For ONNX tracing or something else? As TorchScript is in maintenance mode, I propose avoid adding new dtypes there

Reply:
Then why were the other 2 F8 types added?